config/yo/generator-component/app/templates/Component.css

```diff
@@ -0,0 +1,14 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+
+.<%= className %> {
+  padding: 8px;
```

Comment:
What's the default padding for?

Reply:
Nothing, it was just random CSS contents. Hope people don't take it as a requirement to have 8px padding...